src/compiler-debugging.md

```diff
@@ -140,9 +140,10 @@ stack backtrace:
   (~~~ IRRELEVANT PART OF BACKTRACE REMOVED BY ME ~~~)
   36: rustc_driver::run_compiler
              at /home/user/rust/compiler/rustc_driver/src/lib.rs:253
-$ # Cool, now I have a backtrace for the error
 ```
 
+Now I have a backtrace for the error.
```

Comment:
Maybe say something like:

```suggestion
And now I have a backtrace for the error!
```

or something like the original:

```suggestion
Cool, now I have a backtrace for the error!
```

instead? "Now I have a backtrace for the error." seems rather flat.